recipes/cfitsio/all/conanfile.py

```diff
@@ -94,7 +93,7 @@ def _configure_cmake(self):
         self._cmake.definitions["CFITSIO_USE_SSSE3"] = self.options.get_safe("simd_intrinsics") == "ssse3"
         if self.settings.os != "Windows":
             self._cmake.definitions["CFITSIO_USE_BZIP2"] = self.options.with_bzip2
-            self._cmake.definitions["CFITSIO_USE_CURL"] = self.options.with_curl
+            self._cmake.definitions["UseCurl"] = self.options.with_curl
```

Comment:
Why did you rename this option?
To avoid the change of mixing up an *official* cfitio option and a custom, patched-in, option?

Reply:
yes